doc/gui/examples/controls/chat_messages.py

```diff
@@ -12,10 +12,10 @@
 from taipy.gui import Gui, Icon
 
 msgs = [
-    ["1", "msg 1", "Alice"],
-    ["2", "msg From Another unknown User", "Charles"],
-    ["3", "This from the sender User", "taipy"],
-    ["4", "And from another known one", "Alice"],
+    ["1", "msg 1", "Alice", None],
```

Comment:
Do we really need these 'image' values?

Reply:
I guess we need to create a column if we want to support image